crates/tabby/src/serve/mod.rs

```diff
@@ -161,8 +160,13 @@ pub async fn main(config: &Config, args: &ServeArgs) {
     }
 
     info!("Starting server, this might takes a few minutes...");
+
+    // swagger add local server
+    let mut doc = ApiDoc::openapi();
+    add_localhost_server(&mut doc, args.port);
```

Comment:
```suggestion
    let doc = add_localhost_server(APIDoc::openapi(), args.port);
```

Will something like this work?

Reply:
yes, it more clear